static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -588,6 +591,23 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
             return [];
         });
 
+// This is called by individual banner AbTests in their canRun functions
+// TODO - banner testing needs a refactor, as currently both canRun and canShow need to call this
+export const canShowBannerSync = (
+    minArticlesBeforeShowingBanner: number = 3,
+    userCohort: AcquisitionsComponentUserCohort = 'OnlyNonSupporters'
+): boolean => {
+    const userHasSeenEnoughArticles: boolean =
+        getVisitCount() >= minArticlesBeforeShowingBanner;
+    const bannerIsBlockedForEditorialReasons = pageShouldHideReaderRevenue();
```

Comment:
`bannerIsBlockedForEditorialReasons` is much better than `pageShouldHideReaderRevenue` perhaps we should rename the latter in future